src/corecel/math/Algorithms.hh

```diff
@@ -347,6 +348,27 @@ CELER_FORCEINLINE_FUNCTION ForwardIt min_element(ForwardIt first,
 
 //---------------------------------------------------------------------------//
 // Replace/extend <cmath>
+//---------------------------------------------------------------------------//
+//! Generate overloads for a single-argument math function
+#define CELER_WRAP_MATH_FLOAT_DBL_1(PREFIX, FUNC)        \
+    CELER_FORCEINLINE_FUNCTION float FUNC(float value)   \
+    {                                                    \
+        return PREFIX##FUNC##f(value);                   \
+    }                                                    \
```

Comment:
Weird that clang accepts it but GCC doesn't. The original reason I didn't put the `::` in the definition was because I planned to use `celeritas::detail` for one of the implementations, but that ended up not happening. I still need the `PREFIX` because the apple implementation still needs an actually concatenated `__` prefix.

Reply:
Would `CELER_WRAP_MATH_FLOAT_DBL_1(__func)` not work?